udata/templates/home.html

```diff
@@ -57,7 +57,7 @@ <h1 class="page-header">
 
                 <div class="row">
                     <ul class="card-list">
-                        {% for dataset in current_site.settings.home_datasets %}
+                        {% for dataset in current_site.settings.home_datasets|rejectattr('deleted') %}
```

Comment:
I don't get why we have to do that given that we have `current_site.settings.home_datasets.remove(dataset)` above. Is that for the time between the delete action and the propagation of the signal? How much time do we talk about?